bindings/matrix-sdk-ffi/src/client_builder.rs

```diff
@@ -16,6 +16,7 @@ pub struct ClientBuilder {
     username: Option<String>,
     server_name: Option<String>,
     homeserver_url: Option<String>,
+    passphrase: Option<String>,
```

Comment:
Again, let's zeroize this.

Reply:
Used `Zeroizing` as adding a drop to this struct caused all sorts of build errors 🙈